plugin-vidarr/src/main/java/ca/on/oicr/gsi/shesmu/vidarr/VidarrPlugin.java

```diff
@@ -210,9 +270,10 @@ public void store(SubmitAction action, Object value) {
                                                       + "_ "
                                                       + workflow.getVersion()),
                                           String.format(
-                                              "Workflow %s version %s from Vidarr instance %s on target %s.",
+                                              "Workflow %s version %s; inflight %s; from Vidarr instance %s on target %s.",
```

Comment:
Rather than changing the description, you an add an additional parameter to `defineAction` to provide stuff that goes in a table.

```
() -> Stream.of(new Pair<>("Max-in-flight", definer.get().getMaxInFlightMessage(url, workflow.getName())))
```